mTransit/src/providers/google-maps.ts

```diff
@@ -471,6 +488,58 @@ export class GoogleMaps {
     for(let marker of this.markers){
       marker.setMap(null);
     }
+    for(let marker of this.busPath){
+      marker.setMap(null);
+    }
+    
     this.markers.length = 0;   
+    this.busPath.length = 0;       
+  }
+
+  loadShapes(lat: Number, lng: Number){
```

Comment:
For this method, there is commented code on lines 510-512 and the comment on line 502 seems wrong.